qa/rpc-tests/wallet_broadcast.py

```diff
@@ -7,13 +7,17 @@
 from test_framework.util import assert_equal, connect_nodes_bi, start_nodes, stop_nodes, sync_blocks, wait_bitcoinds
 from decimal import Decimal
 
+base_args = [
```

Comment:
```suggestion
BASE_ARGS = [
```
(Or make this a local.)